### PR TITLE
Adding <author> tags to package.xml files

### PR DESCRIPTION
**Pull request by bsb808**
Addresses Issue #147.

The <author> tags are not required, but nice to have.

To test, just re-make and run your favorite dave example.

### CODE REVIEW

File: models/dave_worlds/package.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <name>dave_worlds</name>
   <version>3.1.1</version>
   <description>The dave_worlds package</description>
-
+  <author email="mabel@openrobotics.org">Mabel Zhang</author>
```
Comment (mabelzhang) [inline models/dave_worlds/package.xml:6]: I can't claim authorship here. I only added 1 file to the package very recently - probably why last modified shows me.

File: models/dave_object_models/package.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <name>dave_object_models</name>
   <version>3.1.1</version>
   <description>The dave_object_models package</description>
-
+  <author email="mabel@openrobotics.org">Mabel Zhang</author>
```
Comment (mabelzhang) [inline models/dave_object_models/package.xml:6]: I can't claim authorship here. I only added 1 file to the package very recently - probably why last modified shows me.

File: models/dave_worlds/package.xml (hunk 1 above)
Comment (bsb808) [reply models/dave_worlds/package.xml]: Switched to @dtdavi1 with 7658a50

File: models/dave_object_models/package.xml (hunk 2 above)
Comment (bsb808) [reply models/dave_object_models/package.xml]: Removed author tag with 7658a50

**Review by dtdavi1 — APPROVED** [review]
Everything looks reasonable.  We might consider including multiple author tags in some of the packages (most of the ones that list me as the author actually have more content from other people than from me), but people should be able to add themselves as they change other things easily enough going forward..

**Pull request merged**